proposals/dark_mode_extension_icons.md

```diff
@@ -166,10 +166,10 @@ effectively have a wild-card for `color_schemes`. For example, `["*"]` is valid.
 **Misc**
 1. If the top-level `icon_variants` key is provided, the top level `icons` key
 will be ignored.
-1. `icon_variants` will not cause an error in the event that it is invalid.
+1. `icon_variants` will not cause an error if it's invalid.
```

Comment:
```suggestion
1. `icon_variants` will not cause hard errors if no icon groups are found or specific icon groups have unknown properties. It may cause a hard error if icon groups are not of type object.
```

Reply:
Added suggested change.